dashboard/modules/reporter/reporter_head.py

```diff
@@ -473,6 +570,32 @@ async def cpu_profile(self, req) -> aiohttp.web.Response:
         else:
             return aiohttp.web.HTTPInternalServerError(text=reply.output)
 
+    @routes.get("/worker/memory_profile")
+    async def memory_profile(self, req) -> aiohttp.web.Response:
+        if "ip" in req.query:
+            reporter_stub = self._stubs[req.query["ip"]]
```

Comment:
handle the case stub doesn't exist (and test)

Reply:
will throw error if no stub with matched ip